ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/discovery/internal/PeerDiscoveryController.java

```diff
@@ -195,6 +201,18 @@ private PeerDiscoveryController(
             "type");
   }
 
+  private void onPeerAdded(final Peer peer, final boolean wasAdded) {
+    if (wasAdded) {
+      addToPeerTable(DiscoveryPeer.fromEnode(peer.getEnodeURL()));
```

Comment:
This doesn't look right.  As we [discussed in the issue](https://github.com/hyperledger/besu/issues/560#issuecomment-641348532), we should [bond](https://github.com/hyperledger/besu/blob/0455a8d937541136adc0b079e169db9ea451e9a8/ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/discovery/internal/PeerDiscoveryController.java#L442) to the peer rather than just adding them to our peer table and immediately advertising the node to the network.

Reply:
Added a follow-up ticket to address since this is already merged: https://github.com/hyperledger/besu/issues/1278